antlr-kotlin-runtime/src/commonMain/kotlin/org/antlr/v4/kotlinruntime/ParserRuleContext.kt

```diff
@@ -41,23 +41,33 @@ public open class ParserRuleContext : RuleContext {
     public val EMPTY: ParserRuleContext = ParserRuleContext()
   }
 
+  // This does not exist in the Java runtime
+  public val position: Position?
+    get() {
+      val start = this.start
+      val stop = this.stop
+
+      if (start != null && stop != null) {
+        val endPoint = stop.endPoint()
+
+        if (endPoint != null) {
+          return Position(start.startPoint(), endPoint)
+        }
+      }
+
+      return null
+    }
+
   /**
    * If we are debugging or building a parse tree for a visitor,
    * we need to track all the tokens and rule invocations associated
    * with this rule's context. This is empty for parsing w/o tree constr.
    * operation because we don't have the need to track the details about
    * how we parse this rule.
    */
+  @JvmField
```

Comment:
Why do we need to make this a JvmField? Do we want the outside world to be able to set the field directly?

Reply:
Well, the outside world was able to set that property anyway, even without `@JvmField`. The difference is without the annotation a getter, a setter, and a backing private field would be generated. With the annotation we only have the public backing field.

Applying `@JvmField` aligns the code with the [Java runtime](https://github.com/antlr/antlr4/blob/811b7fda58bd14d7f0abc496b6fd651dfa01ed97/runtime/Java/src/org/antlr/v4/runtime/ParserRuleContext.java#L51).